Samples/iOS-Swift/iOS-Swift/ErrorsViewController.swift

```diff
@@ -84,6 +84,15 @@ class ErrorsViewController: UIViewController {
         fatalError("This is a fatal error. Oh no 😬.")
     }
 
+    @IBAction func throwFatalDuplicateKeyError(_ sender: Any) {
```

Comment:
`l`: It would be helpful to have more explanation how this test case works, e.g. why does it cause a Duplicate Key error instead of just updating the existing value?
I believe explaining the sample cases could help us in the future to find similar cases for reported issues